source/h1_connection.c

```diff
@@ -182,6 +182,10 @@ struct h1_connection {
 
         /* If non-zero, then window_update_task is scheduled */
         size_t window_update_size;
+
+        /* for bug for bug compatibility with L7 proxies. They can send connection:close on successful connect
+         * responses. This should be ignored. */
```

Comment:
Any specific documentation on this behavior that we can reference? or just the stackoverflow question?

Reply:
I updated the comment.